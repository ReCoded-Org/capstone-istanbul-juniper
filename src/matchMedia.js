/*
 * This solves matchMedia error.
 * Should be deleted when source of error found and fixed.
 * https://jestjs.io/docs/en/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom
 */

window.matchMedia = jest.fn().mockImplementation((query) => ({
  matches: false,
  media: query,
  onchange: null,
  addListener: jest.fn(), // deprecated
  removeListener: jest.fn(), // deprecated
  addEventListener: jest.fn(),
  removeEventListener: jest.fn(),
  dispatchEvent: jest.fn(),
}));
