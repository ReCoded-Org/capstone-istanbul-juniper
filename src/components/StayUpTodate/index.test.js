import React from "react";
import "../../matchMedia";
import renderer from "react-test-renderer";
import StayUpToDate from ".";

it("StayUpToDate renders correctly", () => {
  const tree = renderer.create(<StayUpToDate />).toJSON();
  expect(tree).toMatchSnapshot();
});
