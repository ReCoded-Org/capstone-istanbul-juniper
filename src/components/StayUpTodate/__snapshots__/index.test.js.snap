// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StayUpToDate renders correctly 1`] = `
<div
  className="stayUpToDate"
>
  <h1
    className="ant-typography"
    style={
      Object {
        "WebkitLineClamp": null,
      }
    }
  >
    <span
      className="stayUpToDate__title"
    >
      home.stayUpToDate
    </span>
  </h1>
  <div
    className="ant-row stayUpToDate__twitterContent"
    style={Object {}}
  >
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
    <div
      className="ant-col stayUpToDate__twitterContent__col ant-col-xs-24 ant-col-lg-12"
      style={Object {}}
    >
      <a
        rel="noopener noreferrer"
        target="_blank"
      >
        <img
          className="twitterContent__col__image"
        />
      </a>
    </div>
  </div>
</div>
`;
